packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton/AddressInfoDropdown.tsx

```diff
@@ -55,10 +55,7 @@ export const AddressInfoDropdown = ({
           </span>
           <ChevronDownIcon className="h-6 w-4 ml-2 sm:ml-0" />
         </summary>
-        <ul
-          tabIndex={0}
-          className="dropdown-content menu z-[2] p-2 mt-2 shadow-center shadow-accent bg-base-200 rounded-box gap-1"
-        >
```

Comment:
i actually liked the sober look that it gives without the shadow, goes well with the matte look of the design.. is there any issue if i modified the classes?

Reply:
I prefer to minimize changing predefined components of a framework or codebase when contributing to a repository, when it's not very important.

Reply:
understandable. i only did it since it was purely an independent UI thing, and hence part of the redesigning. but no problem, i will revert it to its initial state.